src/main/java/com/firebolt/jdbc/resultset/FireboltResultSet.java

```diff
@@ -85,9 +86,10 @@ public class FireboltResultSet extends JdbcBase implements ResultSet {
 
 	private String lastReadValue = null;
 
+	@SuppressWarnings("java:S2139") // TODO: Exceptions should be either logged or rethrown but not both
 	public FireboltResultSet(InputStream is, String tableName, String dbName, int bufferSize, boolean isCompressed,
 							 FireboltStatement statement, boolean logResultSet) throws SQLException {
-		log.debug("Creating resultSet...");
+		log.fine("Creating resultSet...");
 		this.statement = statement;
```

Comment:
Why do we use `log.fine` here but `log.log(Level.FINE, ...` elsewhere? Let's make this consistent. Personally I like `log.fine` syntax better as it's less verbose.